tree-sitter-stack-graphs/Cargo.toml

```diff
@@ -69,12 +69,13 @@ regex = "1"
 rust-ini = "0.18"
 serde_json = { version="1.0", optional=true }
 sha1 = { version="0.10", optional=true }
-stack-graphs = { version=">=0.11, <=0.12", path="../stack-graphs" }
+stack-graphs = { version="0.12", path="../stack-graphs" }
 thiserror = "1.0"
 time = { version = "0.3", optional = true }
 tokio = { version = "1.26", optional = true, features = ["io-std", "rt", "rt-multi-thread"] }
 tower-lsp = { version = "0.19", optional = true }
-tree-sitter = ">= 0.19"
+tree-sitter = "0.20" # keep the same minor version as the tree-sitter dependency
+                     # of tree-sitter-graph to prevent install problems
```

Comment:
This seems unfortunate, since it should all work with 0.19 as well.  But it looks like several of the grammars have moved away from inclusive version constraints, so I guess this is the way :shrug:   Maybe we can revert back to the inclusive constraint once https://github.com/tree-sitter/tree-sitter/pull/3069 lands.

Reply:
I don't think this still worked, since tree-sitter-graph has a hard minimum of tree-sitter@0.20.3.